<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>517</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuCreate_line">
     <property name="title">
      <string>Create (line)?</string>
     </property>
     <addaction name="actionOne_To_One"/>
     <addaction name="actionOne_To_Many"/>
     <addaction name="actionMany_To_Many"/>
     <addaction name="actionAggregate"/>
    </widget>
    <addaction name="actionCreate_Table"/>
    <addaction name="menuCreate_line"/>
    <addaction name="separator"/>
    <addaction name="actionEdit_Table"/>
    <addaction name="actionEdit_line"/>
    <addaction name="separator"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionTo_PDM"/>
    <addaction name="actionScript"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionUser_Manual"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionTo_PDM"/>
   <addaction name="actionScript"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPointer"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionCreate_Table"/>
   <addaction name="separator"/>
   <addaction name="actionOne_To_One"/>
   <addaction name="actionOne_To_Many"/>
   <addaction name="actionMany_To_Many"/>
   <addaction name="actionAggregate"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCreate_Table">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/create_entity</normaloff>:/create_entity</iconset>
   </property>
   <property name="text">
    <string>Create Table</string>
   </property>
  </action>
  <action name="actionEdit_Table">
   <property name="text">
    <string>Edit Table  is needed as seperate action? is it double click thing</string>
   </property>
  </action>
  <action name="actionEdit_line">
   <property name="text">
    <string>Edit (line) ? is this any different if it also a double click thing ?</string>
   </property>
  </action>
  <action name="actionTo_PDM">
   <property name="text">
    <string>To PDM ?</string>
   </property>
  </action>
  <action name="actionScript">
   <property name="text">
    <string>Forward Engineer SQL CREATE Script ?</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings (lang  here)</string>
   </property>
  </action>
  <action name="actionUser_Manual">
   <property name="text">
    <string>User Manual ?</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOne_To_One">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/one_one</normaloff>:/one_one</iconset>
   </property>
   <property name="text">
    <string>One To One</string>
   </property>
  </action>
  <action name="actionOne_To_Many">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/one_many</normaloff>:/one_many</iconset>
   </property>
   <property name="text">
    <string>One To Many</string>
   </property>
  </action>
  <action name="actionMany_To_Many">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/many_many</normaloff>:/many_many</iconset>
   </property>
   <property name="text">
    <string>Many To Many</string>
   </property>
  </action>
  <action name="actionAggregate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/agregate</normaloff>:/agregate</iconset>
   </property>
   <property name="text">
    <string>Aggregate</string>
   </property>
  </action>
  <action name="actionPointer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/curs</normaloff>:/curs</iconset>
   </property>
   <property name="text">
    <string>Pointer</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/remove_entity</normaloff>:/remove_entity</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
